client/signup/steps/p2-get-started/index.jsx

```diff
@@ -0,0 +1,100 @@
+import { Button } from '@wordpress/components';
```

Comment:
Could we split these into internal, external and WP dependencies, please?

Reply:
Done!